Frameworks/Foundation/NSInvocation.mm

```diff
@@ -143,17 +143,17 @@ - (void)setArgument:(void*)buf atIndex:(NSInteger)index {
             // Release old value:
             id oldValue = nil;
             [self getArgument:&oldValue atIndex:index];
-            [(*(id*)buf) retain];
+            [(*(id*)buf)retain];
```

Comment:
yeah I guess so.

---
In reply to: [113569073](https://github.com/Microsoft/WinObjC/pull/2636#discussion_r113569073) [](ancestors = 113569073)